tests/unit/Container/ServiceExtensionsTest.php

```diff
@@ -11,6 +11,8 @@ class ServiceExtensionsTest extends TestCase
 {
     /**
      * @test
+     *
+     * @phpstan-type TestObject object{ count: integer }&\stdClass
```

Comment:
```suggestion
     * @phpstan-type TestObject object{count: int}&\stdClass
```

Did you mean to also reference the `TestObject` type in the annotations further down? 🙂